<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShamelaSearchWidget</class>
 <widget class="QWidget" name="ShamelaSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>عبارات البحث:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="SearchQueryWidget" name="searchQueryWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSearchFilter">
     <property name="title">
      <string>مجال البحث</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeView" name="treeViewBooks">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="animated">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>تصفية اللائحة:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="FilterLineEdit" name="lineFilter">
          <property name="pixmap" stdset="0">
           <pixmap resource="resources.qrc">:/data/images/magnifier.png</pixmap>
          </property>
          <property name="extraPadding" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetSelectedFilter" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widgetSearchTools" native="true">
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 187, 187, 255));}
QPushButton{border: 1px solid rgb(173, 173, 173);padding:2px;}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushSelectAll">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>اختيار الكل</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushUnSelectAll">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>الغاء الكل</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushSelectVisible">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>اختيار الكتب الظاهرة</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushUnselectVisible">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>الغاء الكتب الظاهرة</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushExpandTree">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>عرض الشجرة</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushCollapseTree">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>ضغط الشجرة</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>بحث في:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="indent">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboSearchIn">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>المتن  والحاشية</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>المتن  فقط</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>الحاشية فقط</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>ترتيب النتائج حسب:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboSortBy">
       <item>
        <property name="text">
         <string>التطابق</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>الكتاب ثم التطابق</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>كتاب ثم ترتيب الصفحات</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>وفاة المؤلف ثم التطابق</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>وفاة المؤلف ثم الكتاب</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushSearch">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>85</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>بحث</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>pushSearch</zorder>
  <zorder>groupSearchFilter</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchQueryWidget</class>
   <extends>QWidget</extends>
   <header>searchquerywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>filterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
